### ISSUE TITLE
memcalc branch fails rspec

### ISSUE THREAD

**Issue by glyn** [cloudfoundry/java-buildpack#374]
At 0f39e3eeed24bea46e1877a, `bundle exec rake` fails with:
```
Failures:

  1) compile script returns zero if success
     Failure/Error: run("bin/compile #{app_dir} #{app_dir + '.cache'}") { |status| expect(status).to be_success }
       expected `#<Process::Status: pid 74148 exit 1>.success?` to return true, got false
     # ./spec/integration_helper.rb:52:in `block in run'
     # ./spec/integration_helper.rb:50:in `run'

Finished in 1 minute 36.2 seconds (files took 0.57851 seconds to load)
715 examples, 1 failure
```

**Comment by glyn** [comment]
This _may_ be related to issue #375.

**Comment by nebhale** [comment]
Resolved by c45b7f3
